Simplex Trees

olynch opened the pull request.

algebraicjuliabot committed:
Set version to 0.6.4

olynch committed:
started work on simplicial complexes

olynch committed:
Added basic simplicial complexes, vendored trie implementation

Co-authored-by: Kevin Carlson <kevin@topos.institute>

olynch committed:
fixed union and added tests

lukem12345 committed:
3D Differential Operators (#60)

Commit:
2D subdivision, mainly

Commit:
GeometricMap constructor for barycentric subdivision

Commit:
primal vector field migrations with dumb subdivision

KevinDCarlson committed:
Making the dual extraction/subdivision functionality less badly-written

KevinDCarlson committed:
Remove old subdivision function that doesn't go through dual complexes

KevinDCarlson committed:
geometric map from a subdivision updated

KevinDCarlson committed:
little cleanup

lukem12345 committed:
Add GeometryBasics to docs/Project.toml

lukem12345 committed:
Add ACSets to docs/Project.toml

KevinDCarlson committed:
You got you a category of simplicial complexes here

KevinDCarlson committed:
Implementing subdivision functions for all DeltaSet types.

Changes "D_" to "dual_" in DualComplex schemas.

There's some handedness bug still in the comparison between this and
the `subdivide` function.

jpfairbanks commented:
#117 contains many of these commits.

lukem12345 commented:
Potential block by https://github.com/AlgebraicJulia/GATlab.jl/pull/201

https://github.com/AlgebraicJulia/ACSets.jl/pull/174

lukem12345 commented:
@copilot Resolve conflicts.

Copilot committed:
Merge origin/main into multiscale: resolve conflicts

Co-authored-by: lukem12345 <70283489+lukem12345@users.noreply.github.com>

Copilot committed:
Fix import conflicts in SimplicialComplexes after merge with main

Agent-Logs-Url: https://github.com/AlgebraicJulia/CombinatorialSpaces.jl/sessions/2f8f92d4-1abb-4de1-b57d-c8bc2f64e74b

Co-authored-by: lukem12345 <70283489+lukem12345@users.noreply.github.com>
